js.yml

```
@@ -0,0 +1,298 @@
+🟢 1. Basics of Programming & JavaScript Introduction
+
+What is JavaScript
+
+How JS runs (browser, Node.js)
+
+Setting up environment (VS Code + Browser Console)
+
+Writing your first JS program (console.log("Hello World"))
+
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (3 > 2)

(empty-lines)